sdk/extensions/Microsoft.Extensions.Azure/src/Internal/ClientFactory.cs

```diff
@@ -179,11 +179,11 @@ internal static TokenCredential CreateCredential(IConfiguration configuration)
                 throw new ArgumentException("For workload identity, 'tenantId', 'clientId', and 'tokenFilePath' must be specified via environment variables or the configuration.");
             }
 
-            if (string.Equals(credentialType, "azurepipelinescredential", StringComparison.OrdinalIgnoreCase))
+            if (string.Equals(credentialType, "azurepipelines", StringComparison.OrdinalIgnoreCase))
```

Comment:
@scottaddie told me that I needed to fix these because I'm a bad person and should feel bad.

Reply:
For the record, the exact words I used weren't that kind.